bridge/client/styles.scss

```diff
@@ -38,6 +38,11 @@ ktb-root {
   display: flex;
   flex-direction: column;
   place-content: stretch stretch;
+
+  .dt-icon {
+    height: 20px;
+    width: 20px;
```

Comment:
This change affects all icons. Be careful here. It also affects menu icons on the left:
![image](https://user-images.githubusercontent.com/11599148/145034245-d0cd024f-ea0d-4fd2-af6f-208e2164d218.png)
and icons like in the service screen
before:
![image](https://user-images.githubusercontent.com/11599148/145034695-1ad72e9f-127c-4e28-b00f-8a7b6fa35273.png)

now:
![image](https://user-images.githubusercontent.com/11599148/145034725-d99cac28-fed5-45db-ba83-bc77ca9ac97a.png)

maybe there are more places